backend/app/modules/imagefetcher/FetcherActor.scala

```diff
@@ -58,13 +58,15 @@ class FetcherActor(ws: WSClient, serverUrl: String) extends Actor {
         .withQueryString(params:_*)
         .get()
 
+    val currentSender = sender
```

Comment:
Yes, sender refer to the sender of the last message receive by the actor, therefore, access it from a future can lead to an error (access to the sender of the next message instead)